README.md

```diff
@@ -91,16 +91,29 @@ address=":8000"
 
 ```
 $ docker run -e BURROW_BACKEND=http://your-burrow-domain joway/burrow-dashboard:latest
-
 ```
 
 ##### npm
 
 ```
 $ vim src/apis.js // change `baseApi` variable to http://your-burrow-domain
 
+$ npm install
 $ npm run dev
+```
+
+###### local docker build
+
+```
+$ npm install
+
+$ npm npm run build
+
+$ docker build .
```

Comment:
you can set docker tag is by `docker build . -t burrow-dashboard:v2` and just use docker run -e xxx burrow-dashboard:v2